ui-core/src/participant/dashboard/Alert.tsx

```diff
@@ -0,0 +1,89 @@
+import { IconDefinition } from '@fortawesome/fontawesome-svg-core'
+import { faCheck, faCircleInfo, faTriangleExclamation, faXmark } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import classNames from 'classnames'
+import React from 'react'
+import { InlineMarkdown } from '../landing/Markdown'
+
+export type AlertLevel =
+  | 'primary'
+  | 'info'
+  | 'success'
+  | 'warning'
+  | 'danger'
+
+export type AlertTrigger =
+  'NO_ACTIVITIES_REMAIN' |
+  'WELCOME' |
+  'STUDY_ALREADY_ENROLLED'
+
+export type ParticipantDashboardAlert = {
+  id?: string
+  title: string
+  detail?: string
+  type: AlertLevel
+  trigger: AlertTrigger
+}
+
+export const alertDefaults: Record<AlertTrigger, ParticipantDashboardAlert> = {
+  'NO_ACTIVITIES_REMAIN': {
+    title: 'All activities complete',
+    detail: 'You have completed all activities for this study. We will notify you ' +
```

Comment:
I'm confused about why this is here when are allowing admins to edit this message. Presumably we want to get this from the DB?

Reply:
Yes that is the eventual goal, especially with multi-language support as Devon mentioned.